apps/etl-func/avro/message.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "message",
+  "type": "record",
+  "namespace": "dto",
+  "doc": "Kafka JS schema for cosmos api container 'messages'",
+  "fields": [
+    {
+      "name": "schema_version",
+      "type": "int"
+    },
+    {
+      "name": "op",
+      "type": {
+        "type": "enum",
+        "name": "MessageCrudOperation",
+        "symbols": ["CREATE"]
+      }
+    },
+    {
+      "name": "id",
+      "type": "string"
+    },
+    {
+      "name": "sender_service_id",
+      "type": "string"
+    },
+    {
+      "name": "sender_user_id",
+      "type": "string"
+    },
+    {
+      "name": "is_pending",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "content_subject",
```

Comment:
why are we prefixing some fields with `content_` ?

Reply:
As of today, both the subject and the message content are logically treated as part of the overall 'content' of the message, which is why, in the older schema, it was labeled 'content_subject.' I don't have a strong opinion on this, so if you think we can remove the prefix, I'm fine with that.

Reply:
I removed the prefix [here](https://github.com/pagopa/io-messages/pull/40/commits/1ac5f049c9bec589693559028d0e4307d4135d4e)